### PR TITLE
Refactor: ProposalCard agnostic of NNS

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Refactor the ProposalCard into NnsProposalCard and a NNS agnostic one ProposalCard.

Preparing for adding Voting for SNS projects.

# Changes

* Split current ProposalCard into two: NnsProposalCard and ProposalCard.
* New ProposalCard expects primitive props and renders them in the UI.
* Refactored the rendering of ProposalCard to use KeyValuePair component.
* New NnsProposalCard expects a proposal, extracts the details and renders a ProposalCard.
* Refactor ProposalCountdown to Countdown and pass only the timestamp deadline in seconds to make it agnostic of NNS Proposal.

# Tests

* Tests for both splited components ProposalCard and NnsProposalCard.
* Fix other tests after change in props and renamings.
* Fix warning during test.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

There are no UI changes.

File: frontend/src/lib/components/proposals/ProposalCard.svelte [hunk 1]
```diff
@@ -91,32 +73,34 @@
...
-      margin: 0 var(--padding-2x) 0 0;
+    :global(.value) {
+      color: var(--tertiary);
     }
-
-    color: var(--value-color);
   }
 
-  .card-meta {
-    @include card.meta;
```
Comment (mstrasinskis) [inline frontend/src/lib/components/proposals/ProposalCard.svelte:109]: I think this was the only place where this class is used. Would be great to remove it from gix-components as well

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

File: frontend/src/lib/components/proposals/ProposalCard.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/components/proposals/ProposalCard.svelte:109]: Yes, you're right. I'll do that.

Comment (lmuntaner) [reply frontend/src/lib/components/proposals/ProposalCard.svelte:109]: It is used, but inside a component in gix-components: https://github.com/dfinity/gix-components/blob/main/src/lib/components/Card.svelte#L137

**Pull request merged**